nextflow_schema.json

```diff
@@ -259,20 +259,20 @@
                 "annotation_bakta_complete": {
```

Comment:
`annotation_bakta_singlemode` above has a badly formatting 'modifies too parameers' thing, it's missing a new line

Reply:
That's weird. The code looks exactly like that of the other params. Do you have a screenshot?

Reply:
In my `nf-core launch` it looks alright:
![image](https://github.com/nf-core/funcscan/assets/73216762/1974939b-0cae-4a48-8394-53acf6af1e64)